diagnostic.md

```diff
@@ -32,6 +32,22 @@ format described in the instructions). The page should have a title in the head
 and an unordered list (with three items) inside the body.
 
 <!-- your answer starts here -->
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Welcome to my Website!</title>
+  </head>
+  <body>
+
+  </body>
+</html>
+<h1>Welcome to my Website!</h1>
+<ol>
+<li> Item 1 </li>
+<li> Item 2 </li>
+<li> Item 3 </li>
+</ol>
```

Comment:
Your content should go inside your `<body>` tags. It also asks for an unordered list, not ordered. ie:
```HTML
<!DOCTYPE html>
<html>
	<head>
		<title>My Title</title>
	</head>
	<body>
		<ul>
			<li>Item 1</li>
			<li>Item 2</li>
			<li>Item 3</li>
		</ul>
	</body>
</html>
```